jenkinsfiles/ios_jenkinsfile.groovy

```diff
@@ -84,4 +92,4 @@ node('ios') {
     stage('Archive') {
         archiveArtifacts "build/Debug-iphoneos/${OUTPUT_FILE_NAME}"
```

Comment:
ok, so why the built app is always in the `build/Debug-iphoneos` dir then? Is it because `xcodeBuild` step didn't specify a configuration and it will always using the `Debug` config?